client/verta/verta/operations/monitoring/alert/_entities/alert.py

```diff
@@ -439,6 +467,31 @@ def create(
             last_evaluated_at_millis=time_utils.epoch_millis(last_evaluated_at),
         )
 
+    def _build_summary_query(self):
+        if self._summary:
+            return SummaryQuery(
+                ids=[self._summary.id],
+                names=[self._summary.name],
+                monitored_entities=[self._summary.monitored_entity_id],
+            )
```

Comment:
Side note (I'm likely going to do this in my tasks), it might make sense to also have
```python
elif self._monitored_entity_id:
    return SummaryQuery(
        monitored_entities=[self._monitored_entity_id],
    )
```
since the cron alerter needs something like this, but is doing it in a rather hacky way.